mix.exs

```diff
@@ -33,7 +33,8 @@ defmodule Clickhousex.Mixfile do
       {:jason, "~> 1.0"},
       {:ex_doc, "~> 0.22", only: :dev},
       {:benchee, "~> 1.0", only: [:dev, :test]},
-      {:credo, "~> 1.5", only: :dev}
+      {:credo, "~> 1.5", only: :dev},
+      {:bypass, "~> 1.0", only: [:test]}
```

Comment:
I thought about how to test the http client, and I couldn't come up with a good way to mock it (there is no behaviour). So instead I'm using bypass to mock the server.

Reply:
I think bypass is a great solution here